app/views/user.py

```diff
@@ -44,31 +42,58 @@ def get_all():
     )
 
 
+@bp.route("/get-edit-form/<user_uuid>", methods=["GET"])
+@login_required
+def get_edit_form(user_uuid: str):
+    """htmx reauest"""
+    user: m.User | None = db.session.scalar(m.User.select().where(m.User.uuid == user_uuid))
+    if not user or user.is_deleted:
+        log(log.ERROR, "User not found by id: [%s]", user_uuid)
+        return render_template("toast.html", category="danger", message="User not found"), 404
+    form = f.UserForm(
+        user_uuid=user.uuid,
```

Comment:
maybe try to use f.UserForm(**user.__dict__)? its gonna be more flexible in future